ResourcesCard.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+
+import { CalloutCard } from "component/cards/CalloutCard";
+
+import { sendEvt } from "services/event.service";
+
+import libraryIcon from "./library.png";
+import imgFallback from "./resource_fallback.png";
+
+import { getResources } from "services/resources.service";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = (userId) => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = function (resourceUuid) {
+    const evt = new Object();
+    evt = {
+      event_name: "web_dashboard_card_" + r.id + "_" + r.title + "_tap"
+    };
+    sendEvt(evt);
+    console.log("sending event");
+  };
+
+  // Fetch resources
+  useEffect(async () => {
+    const req = await getResources(userId);
+    setResources(req.resources);
+    let primaryResource = resources[0];
+    let secondaryResources = resources.slice(1, 5);
+    setPrimaryResource(primaryResource);
+    setSecondaryResources(secondaryResources);
+  });
+
+  // Get image of first resource
+  const i = () => {
+    const { image } = primaryResource;
+    if (!primaryResource.image || primaryResource.image === undefined) {
+      return imgFallback;
+    }
+
+    if (image === true) {
```

Comment:
This part is a little confusing. Is the image a boolean or a string with URL of the image?